.github/workflows/publish-rc.yaml

```diff
@@ -0,0 +1,77 @@
+on:
+  issue_comment:
+    types:
+      - created
+
+jobs:
+  publish-rc:
+    # This job only runs for pull request comments
+    name: Publish RC PR comment
+    if: ${{ github.event.issue.pull_request }} && contains(github.event.comment.body, 'publish rc')
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Extract branch name
+        shell: bash
+        run: echo "branch=${GITHUB_HEAD_REF:-${GITHUB_REF#refs/heads/}}" >> $GITHUB_OUTPUT
+        id: extract_branch
+
+      # Since this is an explicit publish, we just set the registry provider explicitly
+      - name: Set Environment
+        shell: bash
+        run: echo "ARTIFACT_REGISTRY_PROVIDER=npm" >> $GITHUB_ENV
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16.x'
+          registry-url: 'https://registry.npmjs.org'
+
+      - name: Install Dependencies
+        run: |
+          yarn install --network-timeout 100000
+          git checkout -- package.json
+
+      - name: Linter
+        run: yarn lint
+
+      - name: Unit tests
+        run: yarn test
+
+      - name: Git config
+        run: |
+          git config --global user.email "${{ secrets.GH_DEPLOY_EMAIL }}"
+          git config --global user.name "${{ secrets.GH_DEPLOY_USER }}"
+
+      - name: Apply version bump (rc)
+        run: npm version prerelease --preid=rc -m "Publishing RC - %s"
+
+      - name: Git push version bump
+        run: git push origin ${{ steps.extract_branch.outputs.branch }} --follow-tags --force
+
+      - name: Build
+        run: yarn build-dist
+
+      - name: License report
+        run: yarn license-report-dist
+
+      - name: Compare artifact package version
+        run: .github/scripts/compare-artifact-package-versions.sh
```

Comment:
We could skip this perhaps. Remnant from an older build approach. If a version already exists, that's fine, the npm publish will fail. Dev will be notified of a failed build and can follow up. Otherwise if for whatever reason the publish was skipped.. dev might be waiting endlessly on that 'new npm version published' email. @charlie-tf thoughts on removing this from all builds?

Reply:
I think that your reasoning is sound. I'd be good with removing it

Reply:
If I remove from all builds, should I also remove [this](https://github.com/terraware/web-components/blob/main/.github/workflows/develop.yml#L61) line?

Reply:
Yes 👍

Reply:
Hmm I think we still might want to keep it. Otherwise it will always appear like the build failed. See the latest "Deployment" run for this PR.

Reply:
Oops..

Reply:
You are right, sorry about that. Yes, best to revert the last commit. Keep things as they were for all builds.